cypress/support/pages/course/pageCourses.js

```diff
@@ -30,7 +30,7 @@ class Courses {
 	static #backToDraftButtonInDotMenuOfTopic =
 		'[data-testid="lesson-card-menu-action-revert-0"]';
 	static #taskCardTitleInCoursePageWithIndex = '[data-testid="task-title-0"]';
-	static #boardCardTitleInCoursePageWithIndex = '[data-testid="board-card-title-0"]';
+	static #boardCardTitleInCoursePageWithIndex = '[data-testid="board-title-0"]';
```

Comment:
if is not part of the test id (i mean "card") then maybe also remove it from the variable name

Reply:
data-testid is changed, so it is like this now, 'card' is removed from it, not sure when it was changed.